apitools/base/py/credentials_lib.py

```diff
@@ -39,9 +39,19 @@
 #
 # pylint: disable=wrong-import-order,ungrouped-imports
 try:
-    from oauth2client import gce, locked_file, multistore_file
+    from oauth2client.contrib import gce
```

Comment:
gce is separate.  We can coordinate the locked_file and multistore_file when we try to bring those in internally (that should be the only place where we see this issue -- github oauth2client is up-to-date).